zboxcore/zboxutil/util.go

```diff
@@ -82,6 +82,15 @@ func GetFullRemotePath(localPath, remotePath string) string {
 	return remotePath
 }
 
+func GetFullRemotePathForStreamUpload(remotePath string) string {
+	if remotePath == "" || strings.HasSuffix(remotePath, "/") {
+		remotePath = strings.TrimRight(remotePath, "/")
+		_, fileName := filepath.Split(remotePath)
+		remotePath = fmt.Sprintf("%s/%s", remotePath, fileName)
+	}
+	return remotePath
+}
```

Comment:
`GetFullRemotePathForStreamUpload(remotePath)` looks very similar to `GetFullRemotePath(remoteParthj, remotePath)`. Do we really need two different functions here?